pipelines/smartseq2_single_sample/configuration.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I would call this options.json since that's what cromwell calls it and that's what we call it elsewhere.

Reply:
changed